G1ANT.Addon.UI/G1ANT.Addon.UI/Api/XPathUIElementBuilder.cs

```diff
@@ -112,22 +117,36 @@ public object Operator(XPathOperator op, object left, object right)
         {
             if (op == XPathOperator.Eq)
             {
-                if (left is AutomationProperty property)
+                CompareFunc func = (elem, index) =>
```

Comment:
split to separate lambda functions depends on "left" type, it has to be as fast as possible cause it is called many times